### PR TITLE
fix: Adds the missing launch icon to the help button

### PR DESCRIPTION
**Pull request by benkolera**
The existing launch svg was not a nice color (it's #D7D7D7, which is the hover background color of the button, lol) to use directly on a default colored button, so I made a new launch svg that is the same color as receive.svg so that it works better with the hover and active states of the button.

### CODE REVIEW

File: desktop/src/Desktop/Frontend.hs [hunk 1]
```diff
@@ -1,14 +1,14 @@
-{-# LANGUAGE ConstraintKinds #-}
```
Comment (benkolera) [inline desktop/src/Desktop/Frontend.hs]: It looked like we used stylish haskell in a different file that I changed earlier today, but this one did not have the autoformat previously applied. Should I have this autoformat turned on or off working on chainweaver?

File: desktop/src/Desktop/Frontend.hs [hunk 2]
```diff
@@ -184,7 +184,9 @@ lockScreen xprv = setupDiv "fullscreen" $ divClass "wrapper" $ setupDiv "splash"
 
     e <- confirmButton (def & uiButtonCfg_type ?~ "submit") "Unlock"
     (_help, restore) <- setupDiv "button-horizontal-group" $ do
-      help <- uiButton def $ text "Help" -- TODO where does this go?
+      help <- uiButton def $ do
+        elAttr "img" ("src" =: static @"img/launch_dark.svg" <> "class" =: "button__text-icon") blank
+        text "Help" -- TODO where does this go?
```
Comment (benkolera) [inline desktop/src/Desktop/Frontend.hs:189]: It's worth noting that the help button is still a placebo button. :slightly_smiling_face:

File: desktop/src/Desktop/Frontend.hs (hunk 1 above)
Comment (tomsmalley) [reply desktop/src/Desktop/Frontend.hs]: off please; the diffs are bad otherwise

Comment (benkolera) [reply desktop/src/Desktop/Frontend.hs]: Fixed, as well as adding in https://github.com/kadena-io/chainweaver/pull/223 to get consistency around the place.

**Pull request merged**